microshift_running_apps/microshift-greenboot-workload-scripts.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="microshift-greenboot-workload-scripts"]
+= Greenboot workload health check scripts
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-greenboot-workload-scripts
+
+toc::[]
+
+If you installed the `microshift-greenboot` and `greenboot-default-health-check` RPM packages, you can also create health check scripts assess the health of your workloads and applications. These additional health check scripts are useful components of software problem checks and automatic system rollbacks. Greenboot health check scripts are helpful on edge devices where direct serviceability is either limited or non-existent.
```

Comment:
Agreed. Since it isn't relevant here, we can remove mention of `greenboot-default-health-check` and let that content reside in the main greenboot docs that are in install.